### PR TITLE
adding cleaning_company_column function

### PR DESCRIPTION
**Pull request by adilkassim**

### CODE REVIEW

File: utils/linkage.py [hunk 1]
```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
             string.append(key)
 
     return " ".join(string)
+
+
+"""
+Module for standardizing the 'company' columnn of the state campaign finance dataset
```
Comment (averyschoen) [inline utils/linkage.py]: this is unnecessary

File: utils/linkage.py [hunk 2]
```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
...
+        company: string of inputted company names
+    Returns:
+        standardized for retired, self employed, and unemployed,
+        or original string if no match or empty string
+
+    >>> cleaning_company_column("Retireed")
+    'Retired'
+    >>> cleaning_company_column("self")
+    'Self Employed'
+    >>> cleaning_company_column("None")
```
Comment (averyschoen) [inline utils/linkage.py:155]: Add a test w/ None/NaN

File: utils/linkage.py [hunk 3]
```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
...
+    if pd.isnull(company):
+        return company
+
+    company_edited = company.lower()
+    company_edited = company_edited.strip()
+    company_edited = company_edited.replace(".", " ")
+    company_edited = company_edited.replace(",", " ")
+    company_edited = company_edited.replace("-", " ")
+
+    if "retire" in company_edited:
```
Comment (averyschoen) [inline utils/linkage.py]: let's think about how we can handle people with "retire" in part of their company name

File: utils/linkage.py [hunk 4]
```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
...
+    'Retired'
+    >>> cleaning_company_column("self")
+    'Self Employed'
+    >>> cleaning_company_column("None")
+    'Unemployed'
+    """
+    if pd.isnull(company):
+        return company
+
+    company_edited = company.lower()
```
Comment (averyschoen) [inline utils/linkage.py]: Let's use a regex to condense this step. Check out this page: https://www.geeksforgeeks.org/python-remove-punctuation-from-string/

File: utils/linkage.py [hunk 5]
```diff
@@ -47,3 +48,54 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
...
+        or company_edited == "none"
+        or company_edited == "not employed"
+    ):
+        return "Unemployed"
+
+    else:
+        return company
+
+
+# Example implementation of the function standardize_company_column for a dataframe
```
Comment (averyschoen) [inline utils/linkage.py]: Let's remove this comment for now

**Review by averyschoen — CHANGES_REQUESTED** [review]
Nice work, can you fix the merge conflict and address the inline comments I left? Like we did with Trevor, we'll have the team take a look at the function to discuss some edge cases.

**Review by averyschoen — APPROVED** [review]
